README.md

```diff
@@ -118,12 +119,18 @@ gcloud run deploy ${SERVICE_NAME} \
   --container=${TCPDUMP_SIDECAR_NAME} \
   --image=${TCPDUMP_IMAGE_URI} \
   --cpu=1 --memory=1G \
-  --set-env-vars="PCAP_IFACE=${PCAP_IFACE},PCAP_GCS_BUCKET=${PCAP_GCS_BUCKET},PCAP_FILTER=${PCAP_FILTER},PCAP_JSON_LOG=${PCAP_JSON_LOG} \
+  --set-env-vars="PCAP_IFACE=${PCAP_IFACE},PCAP_GCS_BUCKET=${PCAP_GCS_BUCKET},PCAP_FILTER=${PCAP_FILTER},PCAP_JSON_LOG=${PCAP_JSON_LOG}"
 ```
 
 > See the full list of available flags for `gcloud run deploy` at https://cloud.google.com/sdk/gcloud/reference/run/deploy
 
-3. All containers need to depend on the `tcpdump` sidecar, but this configuration is not available via gcloud due to needing to configure healthchecks for the sidecar container. To make all containers depend on the `tcpdump` sidecar, edit the Cloud Run service via the Cloud Console and make all other containers depend on the `tcpdump` sidecar and add the following TCP startup probe healthcheck to the `tcpdump` sidecar:
+## Final setup
+
+### Configure tcpdump sidecar healthchecks
+
+1. All containers need to depend on the `tcpdump` sidecar, but this configuration is not available via gcloud due to needing to configure healthchecks for the sidecar container. To make all containers depend on the `tcpdump` sidecar, edit the Cloud Run service via the Cloud Console and make all other containers depend on the `tcpdump` sidecar.
```

Comment:
Ideally this should be more of a recommendation then an obligation.

The reason we tell customers this is to catch issues that happen on startup as this will ensure the sidecar is up before the main container.

also tcpdump sidecar should be pcap sidecar

Not things we need to fix in this PR, but things we can tackle in a future one if you don't want to.